### ISSUE TITLE
BRAKER manual confusion about --UTR and --addUTR

### ISSUE THREAD

**Issue by krabapple** [Gaius-Augustus/BRAKER#370]
I'm copying  the relevant manual text below for ease of reference.  Comments describing my confusion are added in bold in square brackets

//

### --UTR=on
Generate UTR training examples for AUGUSTUS from RNA-Seq coverage information, train AUGUSTUS UTR parameters and predict genes with AUGUSTUS and UTRs, including coverage information for RNA-Seq as evidence. This flag only works if --softmasking is also enabled. _This is an experimental feature!_
**[I understand this to mean, --UTR  is for predicting UTRs as part of  a standard BRAKER run]**

**[However, this next line seems to be about *adding* UTRs to an existing output -- a function I thought --addUTR was for]**
If you performed a BRAKER run without --UTR=on, you can add UTR parameter training and gene prediction with UTR parameters (and only RNA-Seq hints) with the following command:
```

braker.pl --genome=../genome.fa --addUTR=on --softmasking \
    --bam=../RNAseq.bam --workingdir=$wd \
    --AUGUSTUS_hints_preds=augustus.hints.gtf \
    --cores=8 --skipAllTraining --species=somespecies
```
**[The command above is _identical_ to the --addUTR command shown below]**

Modify augustus.hints.gtf to point to the AUGUSTUS predictions with hints from previous BRAKER run; modify flaning_DNA value to the flanking region from the log file of your previous BRAKER run; modify some_new_working_directory to the location where BRAKER should store results of the additional BRAKER run; modify somespecies to the species name used in your previous BRAKER run.
**['flaning_DNA' is a typo for --flanking_DNA  but in any case I do not see the option included in either the command for --UTR above or --addUTR below]**

### --addUTR=on
Add UTRs from RNA-Seq converage information to AUGUSTUS gene predictions using GUSHR. No training of UTR parameters and no gene prediction with UTR parameters is performed.

If you performed a BRAKER run without --addUTR=on, you can add UTRs results of a previous BRAKER run with the following command:
```

braker.pl --genome=../genome.fa --addUTR=on --softmasking \
    --bam=../RNAseq.bam --workingdir=$wd \
    --AUGUSTUS_hints_preds=augustus.hints.gtf --cores=8 \
    --skipAllTraining --species=somespecies
```
Modify augustus.hints.gtf to point to the AUGUSTUS predictions with hints from previous BRAKER run; modify some_new_working_directory to the location where BRAKER should store results of the additional BRAKER run; this run will not modify AUGUSTUS parameters. We recommend that you specify the original species of the original run with --species=somespecies. Otherwise, BRAKER will create an unneeded species parameters directory Sp_*.
**[is it crucial to use the original species of the original run, or does it not really matter, if space for the uneeded directory is not an issue?]**
//

** In sum  I am confused as to when to use  --UTR versus --addUTR, and which one requires --flanking_DNA, and when the original species is required**

**Comment by tomasbruna** [comment]
I agree that this section is a bit confusing, it seems to be a mixture of the old and new approaches to UTRs. @KatharinaHoff, can you take a look?

**Comment by KatharinaHoff** [comment]
There's only one approach in BRAKER4: https://github.com/Gaius-Augustus/BRAKER4
